src/firefly/js/templates/fireflyviewer/DefaultLandingPage.jsx

```diff
@@ -21,6 +22,9 @@ export function DefaultLandingPage() {
 
     return (
         <Sheet className='ff-ResultsPanel-StandardView' sx={{width: 1, height: 1}}>
+            {showLandingHelp && <AppHint hintText={'Choose a search from the tabs menu'} sx={{left: 200 + (bannerMiddleStyle?.marginLeft ?? 0)}}/>}
+            {haveBgJobs && <AppHint hintText={'Load job results from background monitor'} sx={{right: 8}} tipPlacement='end'/>}
```

Comment:
I would not rely on `bannerMiddleStyle` for positioning.  It was a bad choice to added it and may be removed in the future.  I would use a reasonable offset instead.  And, if you were doing slotProps, this could be a prop each app can set.

Reply:
Ideally, I wanted to make AppHint hookable to a particular component (how joy ui tooltip works) but that will require JS position calculation magic because AppHint needs to be created at higher level in DOM (direct descendent of body) and has to watch position of banner tabs, which changes in each app.

I didn't realise I can expose the offset prop and let each app set it. Thanks! Also slots design sounds really good - I'll refactor it accordingly.